gensim/models/atmodel.py

```diff
@@ -76,7 +87,21 @@ def __init__(self, eta, lambda_shape, gamma_shape):
 
 
 def construct_doc2author(corpus, author2doc):
-    """Make a mapping from document IDs to author IDs."""
+    """Make a mapping from document IDs to author IDs.
+
+    Parameters
+    ----------
+    corpus: list of list of str
+        Corpus of documents.
+    author2doc: dict of (str: list of int)
```

Comment:
Should looks like `dict of (str, list of int)`, here and everywhere (for all `dict of ..`)